tests/php/View/SSViewerTest.php

```diff
@@ -2369,7 +2372,7 @@ public function testPrimitivesConvertedToDBFields()
     public function testMe(): void
     {
         $mockArrayData = $this->getMockBuilder(ArrayData::class)->addMethods(['forTemplate'])->getMock();
-        $mockArrayData->expects($this->once())->method('forTemplate');
+        $mockArrayData->expects($this->once())->method('forTemplate')->willReturn('');
```

Comment:
Required to avoid returning `null` which breaks the type safety of `ViewableData::XML_val()`